include/rbs/parserstate.h

```diff
@@ -115,6 +115,11 @@ bool parser_typevar_member(parserstate *state, rbs_constant_id_t id);
  * */
 lexstate *alloc_lexer(rbs_allocator_t *, rbs_string_t string, const rbs_encoding_t *encoding, int start_pos, int end_pos);
 
+/**
+ * Allocate new rbs_location_t object.
+ * */
+rbs_location_t *alloc_location(rbs_allocator_t *allocator, range rg);
```

Comment:
Is there something forcing this to be in `parserstate.h`? It fits more logically in `rbs_location.h`

Reply:
Moved 👍